spec.html

```diff
@@ -21167,6 +21165,23 @@ <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
           1. Perform _classScope_.InitializeBinding(_classBinding_, _F_).
         1. Return _F_.
       </emu-alg>
+
+      <emu-clause id="sec-default-constructor-functions">
+        <h1>Default Constructor Functions</h1>
+        <p>When a Default Constructor Function is called with zero or more arguments which form the rest parameter ..._args_, the following steps are taken:</p>
+        <emu-alg>
+          1. If NewTarget is *undefined*, throw a *TypeError* exception.
+          1. If _F_.[[ConstructorKind]] is ~derived~, then
+            1. NOTE: This branch behaves similarly to `constructor(...args) { super(...args); }`. The most notable distinction is that while the aforementioned ECMAScript source text is susceptible to pollution of `%Array.prototype[@@iterator]`, this code is not.
```

Comment:
adding the missing %; the rest is just a suggestion
```suggestion
            1. NOTE: This branch behaves similarly to `constructor(...args) { super(...args); }`. The most notable distinction is that while the aforementioned ECMAScript source text observably calls `%Array.prototype[@@iterator]%`, this abstract operation does not.
```